draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1236,6 +1236,11 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
+If `submission` is an accepted trust anchor whose certifier is neither an
+accepted trust anchor nor the first element of `chain`, then the log MUST return
```

Comment:
Yes, a self-signed trust anchor is the certifier of itself.  When submitting a self-signed trust anchor in `submission`, the `chain` can be empty.